sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1268,6 +1268,16 @@ public void setEnableDeduplication(final @Nullable Boolean enableDeduplication)
     this.enableDeduplication = enableDeduplication;
   }
 
+  /**
+   * Returns if tracing should be enabled. If tracing is disabled, starting transactions returns
+   * {@link NoOpTransaction}.
+   *
+   * @return if tracing is enabled.
+   */
+  public boolean isTracingEnabled() {
+    return getTracesSampleRate() != null || getTracesSampler() != null;
+  }
```

Comment:
this method could be used here too now: https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java#L67-L70

Reply:
👍